javascript_nodejs/14.nlp-with-dispatch/dialogs/mainDialog/resources/dispatch.lu

```diff
@@ -0,0 +1,96 @@
+
```

Comment:
This file is auto-generated, right? maybe add a comment that this file is a product of dispatch tool

Reply:
No. This is generated by my ludown tool. Added a comment to the dispatch.lu file.